setup.py

```diff
@@ -0,0 +1,35 @@
+""" Script for building the cdtz package. """
+
+from setuptools import setup
+
+try:
+    import pypandoc
+
+    LONG_DESCRIPTION = pypandoc.convert('README.md', 'rst')
+except (ImportError, OSError):
+    # OSError is raised when pandoc is not installed.
+    LONG_DESCRIPTION = ('cdtz is microservice intended to '
```

Comment:
Take the description from README (note that it has got a typo, so the following text was corrected).

```python
    LONG_DESCRIPTION = ('CusDeb Tz is a microservice intended to provide the clients with a '
                        'comprehensive list of time zones.')
```